README.md

```diff
@@ -467,27 +467,24 @@ form +++
                 $0.placeholder = "Tag Name"
             }
         }
-        $0 <<< NameRow() {
```

Comment:
Most of time we want to show a data entry row when a MultiRow section is added with insertion mode.